Logger.java

```diff
@@ -120,6 +121,15 @@ static void logException(Exception e) {
     }
   }
 
+  static String getConfigText() throws IOException {
```

Comment:
Why do we throw out? Maybe handle IO errors inside and return null if they happend?